projects/wave-core-new/src/lib/layers/layer-list/layer-list-menu/layer-list-menu.component.ts

```diff
@@ -0,0 +1,37 @@
+import {Component, OnInit, ChangeDetectionStrategy} from '@angular/core';
+import {LayoutService} from '../../../layout.service';
+import {MapService} from '../../../map/map.service';
+import {ProjectService} from '../../../project/project.service';
+import {Observable} from 'rxjs';
+import {Config} from '../../../config.service';
+
+@Component({
+    selector: 'wave-layer-list-menu',
+    templateUrl: './layer-list-menu.component.html',
+    styleUrls: ['./layer-list-menu.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LayerListMenuComponent implements OnInit {
```

Comment:
do you need onInit here?

Reply:
I removed it f23136c0e44778bc3c94661b388e236402d2f49f